src/main/java/com/ebivariation/contigalias/entities/ScaffoldEntity.java

```diff
@@ -0,0 +1,67 @@
+package com.ebivariation.contigalias.entities;
+
+import javax.persistence.*;
```

Comment:
in our style we expand the imports to avoid ambiguity about where a class comes from.